Masquerade4997Code/src/main/java/org/firstinspires/ftc/teamcode/Robots/MarkOne/Test/SkystoneDetectionTest.java

```diff
@@ -29,13 +30,12 @@ public void runLinearOpMode() throws InterruptedException {
 
         if (!robot.detector.isDetected()) robot.runStoneLeft(hardwareMap);
         else {
-            skystonePosition = robot.detector.getStoneSkystonePosition();
-
+            /*skystonePosition = robot.detector.skystoneDetector.getSkystonePosition();
             if (skystonePosition == null) robot.runStoneLeft(hardwareMap);
-            else if (skystonePosition == DogeDetector.SkystonePosition.LEFT) robot.runStoneLeft(hardwareMap);
-            else if (skystonePosition == DogeDetector.SkystonePosition.RIGHT) robot.runStoneRight(hardwareMap);
-            else if (skystonePosition == DogeDetector.SkystonePosition.MIDDLE) robot.runStoneMiddle(hardwareMap);
-            else robot.runStoneLeft(hardwareMap);
+            else if (skystonePosition == SkystoneDetector.SkystonePosition.LEFT) robot.runStoneLeft(hardwareMap);
+            else if (skystonePosition == SkystoneDetector.SkystonePosition.RIGHT) robot.runStoneRight(hardwareMap);
+            else if (skystonePosition == SkystoneDetector.SkystonePosition.MIDDLE) robot.runStoneMiddle(hardwareMap);
+            else robot.runStoneLeft(hardwareMap);*/
```

Comment:
? Why is this commented out. Isn't that useful?

Reply:
It gave some error and I’m not using it now so when I use it I will I comment it